src/base.html

```diff
@@ -233,10 +233,12 @@
       {% include "partials/hero.html" %}
       {% endblock %}
 
-      <!-- Navigation tabs -->
+      <!-- Navigation tabs (collapsing) -->
       {% block tabs %}
-        {% if "navigation.tabs" in features %}
-          {% include "partials/tabs.html" %}
+        {% if not "navigation.tabs.sticky" in features %}
+          {% if "navigation.tabs" in features %}
+            {% include "partials/tabs.html" %}
+          {% endif %}
```

Comment:
Remind me to remove the bit from our docs about tabs only showing in large viewport.